src/main/java/uk/gov/hmcts/darts/task/runner/impl/AbstractLockableAutomatedTask.java

```diff
@@ -94,6 +94,11 @@ public void run() {
                 if (isManualTask || getLastCronExpression().equals(dbCronExpression)) {
                     if (TRUE.equals(automatedTask.getTaskEnabled())) {
                         logApi.taskStarted(executionId.get(), this.getTaskName());
+
+                        if (AutomatedTaskName.PROCESS_DAILY_LIST_TASK_NAME.getTaskName().equals(this.getTaskName())) {
```

Comment:
Sorry Ricardo, not this task. It's the one labelled 'DAILY_LIST_HOUSEKEEPING_TASK_NAME'

Reply:
updated